src/routes/logout/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { SessionContext } from '../../context/session_context';
+
+class LogoutPage extends React.Component<{}, { session: SessionContext }> {
+
+  componentDidMount(): void {
+    const { session } = this.props;
+    session.logout().then(
+      // not my favorite solution... but SSO seems to need a second after /logout is hit before it actually "takes"...
+      setTimeout(() => {
```

Comment:
Interesting - I'm noticing this in several places where we are using setters....